go.work.sum

```diff
@@ -0,0 +1,633 @@
+cloud.google.com/go/accessapproval v1.7.4 h1:ZvLvJ952zK8pFHINjpMBY5k7LTAp/6pBf50RDMRgBUI=
```

Comment:
How do these get updated? Is it possible that someone will bump a version but forget this file?

Reply:
go.work.sum is updated automatically in the same way go.sum is